plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/shipping/shipping-collaterals.tsx

```diff
@@ -0,0 +1,16 @@
+export const ShippingCollaterals = ( {
```

Comment:
I think this component is redundant

Reply:
https://github.com/woocommerce/woocommerce/pull/52313/commits/ea527b4ebbb33c86fb2801aab7f373d646e92dd8